apps/web-giddh/src/assets/styles/base/_global.scss

```
@@ -6008,8 +6008,10 @@ hamburger-menu {
 }
 
 .table {
+    $bg-color: var(--table-background-color);
     width: 100%;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)